pandas/io/json/_normalize.py

```diff
@@ -49,14 +50,19 @@ def nested_to_record(
 
         .. versionadded:: 0.20.0
 
-    level: int, optional, default: 0
+    level : int, optional, default: 0
         The number of levels in the json string.
 
-    max_level: int, optional, default: None
+    max_level : int, optional, default: None
         The max depth to normalize.
 
         .. versionadded:: 0.25.0
 
+    use_key : callable, optional, default: None
```

Comment:
Can you match the signature of usecols in the CSV world? Should accept a string, list of strings or callable